packages/notebook/src/tokens.ts

```diff
@@ -130,3 +138,62 @@ export interface INotebookTracker extends IWidgetTracker<NotebookPanel> {
    */
   readonly selectionChanged: ISignal<this, void>;
 }
+
+export interface IExecutionOptions {
```

Comment:
So if an extension wants to implement both `INotebookCellExecutor` and `IConsoleCellExecutor` this will be a slightly annoyance for explicit typing because one cannot import to `IExecutionOptions` objects without renaming them in the import. Some options include renaming it to `INotebookCellExecutionOptions`/`IConsoleCellExecutionOptions` or creating a namespace for each `INotebookCellExecutor` and `IConsoleCellExecutor` and just calling it `IOptions`. What do you think?

```suggestion
/**
* Execution options for notebook cell executor.
*/
export interface IExecutionOptions {
```